roles/baseline/tasks/cert-manager.yaml

```diff
@@ -11,7 +11,7 @@
     create_namespace: true
     wait: true
   when:
-    - CERT_MANAGER_ENABLED
+    - (V4_CFG_TLS_GENERATOR | default('default_value', 'openssl')) == 'cert-manager'
```

Comment:
Reasoning behind the extra logic here? Is there a reason for the default? You can have a when check to see if the variable is set. There are lots of examples in the DaC code base.

Reply:
Missed this comment in my inbox. You are right, checking if defined + value=cert-manager afterwards is a cleaner. 

Swapping this to
```
    -  V4_CFG_TLS_GENERATOR is defined
    -  V4_CFG_TLS_GENERATOR == 'cert-manager'
```